javascript/URDFLoader.js

```diff
@@ -94,7 +94,10 @@ class URDFLoader {
 
         // Check if a full URI is specified before
         // prepending the package info
+
         let path = urdf;
+
+        // If path is relative
```

Comment:
Here I was not sure what the if case does. What does it check? (Is my assumption correct?)